.github/stale.yml

```diff
@@ -0,0 +1,20 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 30
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 7
+# Issues with these labels will never be considered stale
+exemptLabels:
+  - bug
+  - "MERGE ME"
+# Label to use when marking an issue as stale
+staleLabel: stale
+# Comment to post when marking an issue as stale. Set to `false` to disable
+markComment: >
+  This issue has been automatically marked as idle and stale because it hasn't
+  had any recent activity. It will be automtically closed if no further activity
+  occurs. If you think this is wrong, or the problem still persists, just pop
+  a reply in the comments and @staabm or @Arkni will (try!) to follow up.
```

Comment:
please dont mention us... the bot will spam us with notifications ;-)

Reply:
Haha, I forgot why I put them in there.

I will replace them with `one of the maintainers`